lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -1527,57 +1528,66 @@ def show_plugins # :nodoc:
             print(tbl.to_s)
           end
 
-          def show_module_set(type, module_set, regex = nil, minrank = nil, opts = nil) # :nodoc:
-            count = -1
+          # @param [table_name] used to name table
+          # @param [module_filter] this will either be a modules fullname, or it will be an Array(show payloads/encoders)
+          # or a Hash(show favorites) containing fullname
+          # @param [compatible_mod] handles logic for if there is an active module when the
+          # `show` command is run
+          #
+          # Handles the filtering of modules that will be generated into a table
+          def show_module_metadata(table_name, module_filter)
+            count = 0
```

Comment:
```suggestion
            count = -1
```

Metasploit's tables start from 0.

Master:
```
msf6 > show encoders

Encoders
========

   #   Name                          Disclosure Date  Rank       Check  Description
   -   ----                          ---------------  ----       -----  -----------
   0   cmd/brace                                      low        No     Bash Brace Expansion Command Encoder
```

This branch:
```
msf6 > show encoders

Encoders
========

   #   Name                                  Disclosure Date  Rank       Check  Description
   -   ----                                  ---------------  ----       -----  -----------
   1   encoder/cmd/brace                                      low        No     Bash Brace Expansion Command Encoder
```

Reply:
I didn't realise that and thought this was just an error, makes sense now as to why it was beginning at `-1` 👍 

Will get that added back in.

Reply:
Looked into this a bit more, sessions start at 1, but these tables start at 0 after this PR was introduced
https://github.com/rapid7/metasploit-framework/pull/11819